src/components/organisms/HelmRepoPane/HelmChartDetails/HelmChartDetails.tsx

```diff
@@ -0,0 +1,188 @@
+import {useCallback, useMemo, useState} from 'react';
+import {useAsync} from 'react-use';
+
+import {Dropdown} from 'antd';
+
+import {CloudDownloadOutlined, DownOutlined} from '@ant-design/icons';
+
+import {first} from 'lodash';
+import {Tab} from 'rc-tabs/lib/interface';
+
+import {kubeConfigContextSelector} from '@redux/appConfig';
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {setHelmPaneChartDetailsTab, setHelmPaneSelectedChart} from '@redux/reducers/ui';
+import {installHelmRepoChart} from '@redux/thunks/InstallHelmRepoChart';
+
+import {HelmChartModalConfirmWithNamespaceSelect} from '@components/molecules';
+
+import {Icon} from '@monokle/components';
+import {HelmChartDetailsTab} from '@shared/models/ui';
+import {trackEvent} from '@shared/utils';
+import {runCommandInMainThread, searchHelmRepoCommand} from '@shared/utils/commands';
+
+import HelmInfo from './HelmChartTabs/HelmInfo';
+import HelmTemplate from './HelmChartTabs/HelmTemplate';
+import HelmValues from './HelmChartTabs/HelmValues';
+import HelmReadme from './HelmChartTabs/HemlReadme';
+import PullHelmChartModal from './PullHelmChartModal';
+
+import * as S from './styled';
+
+const createTabItems = (chartName: string): Tab[] => [
+  {
+    key: 'info',
+    label: 'Info',
+    children: <HelmInfo chartName={chartName} />,
+  },
+  {
+    key: 'templates',
+    label: 'Templates',
```

Comment:
helm template actually performs a dry-run of the Helm Chart - I suggest we 
- rename the tab to "Dry-run Output"
- move it to be after "Default Values"